### PR TITLE
fix steamID url, improve !trades & !trade messages

### PR DESCRIPTION
**Pull request by rocanruca**
-Fixed the steamID in the url for steamProfile, backpackTF and steamREP (this problem is only when using !trade command, it show admin steamID instead of trade partner steamID).
-Added some improvements in messages for !trade & !trades commands.

### CODE REVIEW

File: src/classes/Commands.ts [hunk 1]
```diff
@@ -2247,7 +2247,8 @@ export = class Commands {
 
             reply += `\n- Offer #${offer.id} from ${
                 offer.data.partner
-            } (reason: ${offer.data.action.meta.uniqueReasons.join(', ')})`;
+            } (reason: ${offer.data.action.meta.uniqueReasons.join(', ')})
+            ⚠️ Send "!trade ${offer.id}" for more details.\n `;
```
Comment (idinium96) [inline src/classes/Commands.ts:2251]: please separate this, and why you put `\n` at the end of it?
```
            reply += `\n- Offer #${offer.id} from ${
                offer.data.partner
            } (reason: ${offer.data.action.meta.uniqueReasons.join(', ')})` +
                '⚠️ Send "!trade ${offer.id}" for more details.';
```

File: src/classes/Commands.ts [hunk 2]
```diff
@@ -2324,8 +2325,9 @@ export = class Commands {
                     : ')');
         }
 
-        const links = (this.bot.handler as MyHandler).tradePartnerLinks(steamID.toString());
-        reply += `\n\nSteam: ${links.steamProfile}\nBackpack.tf: ${links.backpackTF}\nSteamREP: ${links.steamREP}`;
+        const links = (this.bot.handler as MyHandler).tradePartnerLinks(offerData.partner.toString());
+        reply += `\n\nSteam: ${links.steamProfile}\nBackpack.tf: ${links.backpackTF}\nSteamREP: ${links.steamREP} 
+        \n⚠️ Send "!accept ${offerId}" to accept or "!decline ${offerId}" to decline this offer.`;
```
Comment (idinium96) [inline src/classes/Commands.ts:2330]: also separate this.

**Review by idinium96 — COMMENTED** [review]
please separate these lines of code. thank you.

File: src/classes/Commands.ts (hunk 1 above)
Comment (rocanruca) [reply src/classes/Commands.ts:2251]: With `\n` at the end
image

Without `\n` at the end
image
I like it more with `\n` at the end

Comment (idinium96) [reply src/classes/Commands.ts]: oh I see.. alright you can keep it 👍

**Pull request merged**